### ISSUE TITLE
Question: a reason why does web browser access to AOAI directly in illustration of sample `3`, `4` and `5`

### ISSUE THREAD

**Issue by horihiro** [Azure-Samples/jp-azureopenai-samples#98]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

Could you please explain a reason why does a web browser access to AOAI directly?

Ex: in [Sample 5](https://github.com/Azure-Samples/jp-azureopenai-samples/blob/main/5.internal-document-search/assets/appcomponents.png).

![image](https://github.com/Azure-Samples/jp-azureopenai-samples/assets/4566555/28d9f9dc-8aaa-40d6-ac94-06fea3a38f65)

It seems that this is just illustration issue.
But if the implemetation is same as the illustration, I think it's better and more secure that all requests from the web browser to AOAI are handled by a web application in App Service.

**Comment by marumaru1019** [comment]
Thank you for pointing this out.
As you indicated, requests from App Service to AOAI are made in Chapter 5 in the current implementation, and we have confirmed that this is what is being implemented as illustrated. In this regard, to further enhance security and integrity, we believe that it would indeed be desirable to process all requests from the web browser to the AOAI through the App Service web application. To this end, we have already made corrections to the implementation in Chapter 5. We also note that the same issues may exist in Chapters 3 and 4, so we will carefully review these chapters and make similar fixes as needed. We intend to make a series of corrections to improve safety and performance.
